packages/labs/src/components/select/multiSelect.tsx

```diff
@@ -79,7 +79,17 @@ export class MultiSelect<T> extends React.Component<IMultiSelectProps<T>, IMulti
     private input: HTMLInputElement;
     private queryList: QueryList<T>;
     private refHandlers = {
-        input: (ref: HTMLInputElement) => (this.input = ref),
+        input: (ref: HTMLInputElement) => {
+            this.input = ref;
+
+            const { tagInputProps = {} } = this.props;
+            const inputProps: HTMLInputProps = tagInputProps.inputProps || {};
+            // can't use safeInvoke cuz inputProps.ref can be `string | function`
+            const refHandler = inputProps.ref;
```

Comment:
@AlexMarvelo why does this file use `.ref` while the one below uses `.inputRef`?

Reply:
Described in PR description, point 2

Reply:
this PR for 2.0 added `TagInput.inputRef` prop:  https://github.com/palantir/blueprint/pull/2034/files#diff-534314697ccd64472fc61bbacae83ae1R34

i'm inclined to wait for 2.0 for this whole thing. is that acceptable?